### DISCUSSION TITLE
How to create offline backup of my anytype data on Android 14?

### DISCUSSION THREAD

**Discussion by blazing-ph0enix** [anyproto/anytype-kotlin#1527] [Q&A]
As the title suggests, I am on android 14 so I can't access android/data and its contents.

How to export or backup anytype data?

**Comment by uburoiubu** [comment]
we need to implement it in the next release - 0.38.x

**Comment by Soukyuu** [comment]
I'm guessing that didn't happen in 0.38.x as I'm on 0.40.19-beta and I still can't seem to find a way to backup my data. That's especially a problem because I currently have no way to bring up my self-hosted any-sync nodes and P2P sync seems to be non-functional...
